setup.cfg

```diff
@@ -1,10 +1,17 @@
+# Since there is overlap in the violations that the different tools check for, it makes sense to quiesce some warnings 
+# in some tools if those warnings in other tools are preferred. This avoids the need to add duplicate lint warnings.
+
 [pycodestyle]
 max-line-length = 119
 
 [flake8]
 max-line-length = 119
-ignore = E722
+ignore = E722,F841
+# E722: pylint is preferred for bare-except
+# F841: pylint is preferred for unused-variable
 
-# pylint
+# pylint honours the [MESSAGES CONTROL] section
 [MESSAGES CONTROL]
-disable=invalid-name,bare-except
+disable=invalid-name,bare-except,broad-except
```

Comment:
So you disable the warnings about except without exception type totally?

Reply:
I have explained this issue several times during these cleanup commits. I am unable to fully test which exceptions might be thrown without a more thorough testing setup. If you know for sure which exceptions may be thrown at each bare/broad except then please fill them in and you can take the abuse for any runtime failures that crop up. We know that we have to fix the disabled checks, we're just not doing it now.

Reply:
I don't request you to add exception types. But I don't like an idea to mask any issue which you cannot resolve immediately. Let lint warn.

Reply:
I understand where you're coming from regarding masking warnings. However, there are exactly three lint warnings that are disabled project-wide: invalid-name, bare-except and broad-except. Invalid name will be fixed with a project-wide refactor of names once the other style and lint issues are fixed, the other two require me to have a better test process.

In the meantime we can use the return status of `fab code_quality` to ensure, as part of our CI and/or git hooks, that we don't regress in terms of code quality. Once we have those lint warnings (that are currently disabled) fixed, the only thing left is the lines that are between 80 and 119 characters, which can be fixed as we go, or when we're slack.

If I were thinking like you are now I would be much more concerned with the three lines worth of lint warnings that I have disabled at the top of the most offending modules. These require serious refactoring in terms of the structure of the module. It cannot be done without a rigorous testing process, I cannot do it personally until I understand the code better. Rest assured, it will be done.